package.json

```diff
@@ -28,14 +28,14 @@
     "grunt-ts": "^6.0.0-beta.19",
     "grunt-typedoc": "^0.2.4",
     "intern": "^4.4.3",
-    "ion-js": "^4.0.0",
-    "jsbi": "3.1.1",
-    "typedoc": "^0.17.3",
+    "ion-js": "^5.2.0",
+    "jsbi": "^4.3.0",
```

Comment:
Is JSBI even required—can we just remove it from the dependencies completely?

Reply:
Yes, we can remove it from the dependencies. There are build errors caused by incompatible dependency versions and I'm looking into it.